openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -523,11 +508,11 @@ OpenAssessment.ResponseView.prototype = {
     fileUpload: function() {
         var view = this;
         var fileUpload = $(".file__upload");
-        fileUpload.addClass("is--disabled");
+        fileUpload.prop('disabled', true);
 
         var handleError = function(errMsg) {
             view.baseView.toggleActionError('upload', errMsg);
-            fileUpload.removeClass("is--disabled");
+            fileUpload.prop('disabled', false);
```

Comment:
It may be that attr/removeAttr was the correct way to do it in older versions of JQuery. I did read something about that here--

http://stackoverflow.com/questions/6048022/jquery-attrdisabled-disabled-not-working-in-chrome